go.mod

```
@@ -0,0 +1,25 @@
+module github.com/mikewurtz/taskman
+
+go 1.24.1
+
+require (
+	github.com/stretchr/testify v1.10.0
+	google.golang.org/grpc v1.71.1
+	google.golang.org/protobuf v1.36.6
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/spf13/pflag v1.0.6 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+require (
```

Suggestion: collapse the indirect blocks into a single block.
```suggestion
	gopkg.in/yaml.v3 v3.0.1 // indirect
```